script.js

```diff
@@ -105,6 +123,10 @@ $(document).on('keypress', function (e) {
   if (e.which == 13) {
     spin(2);
   }
+  // listen on h key and hide and unhide
+  if (e.which == 104) {

```

Comment:
Would it work as the following? And possibly also on enter '\r' possibly?

```suggestion
  if (e.which == 'h') {
```

Reply:
Done